pkg/action/action_handler.go

```diff
@@ -36,11 +37,13 @@ var (
 	turboActionContainerResize     turboActionType = turboActionType{proto.ActionItemDTO_RIGHT_SIZE, proto.EntityDTO_CONTAINER}
 	turboActionContainerPodSuspend turboActionType = turboActionType{proto.ActionItemDTO_SUSPEND, proto.EntityDTO_CONTAINER_POD}
 
-	//turboActionUnbind          turboActionType = "unbind"
+	turboActionMachineProvision turboActionType = turboActionType{proto.ActionItemDTO_PROVISION, proto.EntityDTO_VIRTUAL_MACHINE}
+	turboActionMachineSuspend   turboActionType = turboActionType{proto.ActionItemDTO_SUSPEND, proto.EntityDTO_VIRTUAL_MACHINE}
```

Comment:
if there isn't a valid caClient, then kubeturbo probably shouldn't declare support of the VM scaling actions?

Reply:
We need to handle the situation where the cluster API is enabled after both kubernetes cluster and kubeturbo are deployed.
Which means that kubeturbo should be able to utilize the cluster API being deployed at any point of time without being restarted.

Reply:
For now, I wouldn't suggest to spend too much effort to figure out how to avoid restarting, since restart Kubeturbo is cheap,  it's just a probe and it's stateless. We just need to make sure we don't declare VM scaling action like Cheuk said, if the Cluster API is not there when Kubeturbo starts.